sys-utils/libcmt/Makefile

```diff
@@ -0,0 +1,191 @@
+# paths
+PREFIX = /usr
+TARGET_PREFIX ?= $(PREFIX)
+
+TOOLCHAIN_PREFIX ?= riscv64-linux-gnu-
+TARGET_CC := $(TOOLCHAIN_PREFIX)gcc
+TARGET_AR := $(TOOLCHAIN_PREFIX)ar
+TARGET_CFLAGS := -Wvla -O2 -g -Wall -pedantic -Wextra -Iinclude -Iinclude/libcmt/ioctl
+CFLAGS := -Wvla -O2 -g -Wall -pedantic -Wextra -fsanitize=address,undefined -Iinclude -Iinclude/libcmt/mock
```

Comment:
I hope this sanitizer flags is being disabled for the library that is being released.

Reply:
This is only for `HOST`.